### PR TITLE
added add quiz button to each course

### PR DESCRIPTION
**Pull request by maheshgudi**
Added an add quiz button to each course. 
screenshot from 2016-09-06 12 46 23

### CODE REVIEW

File: yaksh/views.py [hunk 1]
```diff
@@ -246,33 +246,40 @@ def edit_question(request, question_id=None):
                                      context_instance=ci)
 
 @login_required
-def add_quiz(request, quiz_id=None):
+def add_quiz(request, course_id, quiz_id=None):
     """To add a new quiz in the database.
     Create a new quiz and store it."""
     user = request.user
+    course = Course.objects.get(id=course_id)
```
Comment (prathamesh920) [inline yaksh/views.py]: Better if you use get_object_or_404

File: yaksh/templates/yaksh/courses.html [hunk 2]
```diff
@@ -130,22 +134,21 @@
...
                 </div>
             </div>
+            <button class="btn primary"type="button" onClick='location.replace("{{URL_ROOT}}/exam/manage/addquiz/{{course.id}}/");'>Add New Quiz</button>
         </div>
     </div>
     <br><br>
     {% endfor %}
 {% else %}
-    <center><h4> No new Courses allotted </h4></center>
+    <center><h4> No new Courses allotted <br><br></h4></center>
```
Comment (prathamesh920) [inline yaksh/templates/yaksh/courses.html]: Is br tag required here?

File: yaksh/forms.py [hunk 3]
```diff
@@ -153,13 +153,15 @@ class QuizForm(forms.ModelForm):
...
         self.fields['prerequisite'] = forms.ModelChoiceField(
-                queryset=Quiz.objects.filter(course__creator=user))
+                queryset=Quiz.objects.filter(course__creator=user, 
+                                             is_trial=False))
         self.fields['prerequisite'].required = False
         self.fields['course'] = forms.ModelChoiceField(
-                queryset=Course.objects.filter(Q(creator=user)|
-                    Q(teachers=user)).distinct())
+                queryset=Course.objects.filter(id=course_id))
+        self.fields['course'].initial = Course.objects.get(id=course_id)
```
Comment (prathamesh920) [inline yaksh/forms.py]: It would be better if we can lock the initial value, so that user does not change it.

File: yaksh/views.py (hunk 1 above)
Comment (maheshgudi) [inline yaksh/views.py]: resolved

File: yaksh/templates/yaksh/courses.html (hunk 2 above)
Comment (maheshgudi) [inline yaksh/templates/yaksh/courses.html]: resolved

File: yaksh/forms.py (hunk 3 above)
Comment (maheshgudi) [inline yaksh/forms.py]: resolved

**Comment by prathamesh920** [comment]
looks good to me

**Pull request merged**